### DISCUSSION TITLE
SNOMED Code to MSDRG

### DISCUSSION THREAD

**Discussion by yvznmn** [synthetichealth/synthea#1122] [Q&A] [answered]
Is it possible to create data with MSDRG code in Synthea? Data that is populated in encounters, claims tables are SNOMED code. For me to perform my analytic I need MSDRG code that refer SNOMED data.

**Comment by jawalonoski** [comment, marked as answer]
No, it is not possible using Synthea out of the box.

Yes, it is possible if you are willing to do some work.

It is possible to post-process Synthea SNOMED codes into ICD-10 codes, and from there, roll up the ICD-10 codes into MSDRG codes.

To map SNOMED into ICD-10, you will have to use a mapping file. For example, you might use the resources available from the US National Library of Medicine. https://www.nlm.nih.gov/research/umls/mapping_projects/snomedct_to_icd10cm.html 

To map ICD-10 into DRG, use the resources available from https://www.cms.gov/Medicare/Coding/ICD10/ICD-10-MS-DRG-Conversion-Project, specifically the Appendices listed in https://www.cms.gov/Medicare/Coding/ICD10/Downloads/ICD-10-MS-DRG-v32-Definitions-Manual-Text.zip
